README.md

```diff
@@ -1 +1,5 @@
-# cesium-webstorm-plugin
\ No newline at end of file
+# Cesium WebStorm Plugin
```

Comment:
If you want to really kill it, include an animated gif that shows swapping between source and spec files.